src/gateways/AadAuthGateway.ts

```diff
@@ -2,16 +2,31 @@ import {
   ClientCredentialRequest,
   ConfidentialClientApplication,
 } from "@azure/msal-node";
-import { appConfig } from "../appConfig";
+import { NodeAuthOptions } from "@azure/msal-node/dist/config/Configuration";
 import { AuthGateway } from "./interfaces/AuthGateway";
 
 export class AadAuthGateway implements AuthGateway {
+  private config: { auth: NodeAuthOptions; apiId: string };
+
+  public constructor(
+    config = {
+      auth: {
+        clientId: process.env.WEBAPP_CLIENT_ID,
+        authority: `https://login.microsoftonline.com/${process.env.AAD_TENANT_ID}`,
+        clientSecret: process.env.WEBAPP_CLIENT_SECRET,
+      },
+      apiId: process.env.AAD_API_ID,
+    }
+  ) {
+    this.config = config;
+  }
+
```

Comment:
As mentioned previously, allowing injection of config on instantiation for easier testing.